@xen-orchestra/web-core/lib/locales/fr.json

```
@@ -23,6 +23,7 @@
   "alarm-type.unknown": "Type d'alarme inconnu",
   "alarms": "Alarmes",
   "all-good": "Tout va bien !",
+  "all-done": "Tout est fait !",
```

in context of svg, I prefer `Tout est ok !`

```suggestion
  "all-done": "Tout est ok !",
```

```suggestion
  "all-done": "Tout est terminé !",
```